nomic/project.py

```diff
@@ -366,14 +365,15 @@ def __init__(self, atlas_index_id, name, indexed_field, projections):
     def _repr_html_(self):
         return '<br>'.join([d._repr_html_() for d in self.projections])
 
+
 class AtlasProjection:
     '''
-    Manages operations on maps such as text/vector search.
+    Interact and access state of an Atlas Map including text/vector search.
     This class should not be instantiated directly.
     Instead instantiate an AtlasProject and use the project.indices or get_map method to retrieve an AtlasProjection.
```

Comment:
Instead instantiate an AtlasProject and use the project.indices or get_map method to retrieve an AtlasProjection.
should be
    Instead instantiate an AtlasProject and use the project.maps attribute to retrieve an AtlasProjection.